_layouts/base.njk

```diff
@@ -0,0 +1,47 @@
+{% extends "govuk/template.njk" %}
+
+{% block head %}
+  <link rel="stylesheet" href="/assets/govuk.css">
+{% endblock %}
+
+{% block pageTitle %}
+  {{- title if title -}}
+{% endblock %}
+
+{% block header %}
+  <header class="govuk-header" role="banner" data-module="govuk-header">
```

Comment:
Shame there's no way to customise the logo in `govukHeader`. But we can make it customisable in `appHeader` (in the plug-in)!

Reply:
Yeah, maybe! I kinda figure it's ok to switch to just using plain ol HTML for the header, as it's only in a single file so there's less benefit in turning it into a nunjucks macro etc...